packages/docs/stories/components/button.stories.ts

```diff
@@ -1,10 +1,4 @@
-// ---------------------------------------------------------------------
-// 🔒 AUTOGENERATED BY VENDORISM
-// Removing this comment will prevent it from being managed by it.
-// ---------------------------------------------------------------------
-
 /* eslint-disable */
-
 /* eslint-disable import/no-relative-packages */
```

Comment:
Please remove eslint-disable here and for all other stories in the other PRs. As this code is under our control after ejecting, it should fit to our coding guidelines.